aws-events-rule/.gitignore

```diff
@@ -0,0 +1,26 @@
+# macOS
+.DS_Store
+._*
+
+# Maven outputs
+.classpath
+
+# IntelliJ
+*.iml
+.idea
+out.java
+out/
+.settings
+.project
```

Comment:
nit: not sure why its still committing `idea/`  even though I see that in `.gitignore`

Reply:
I just deleted those. I don't know how they got there either...